modules/readme/Makefile

```diff
@@ -29,8 +29,7 @@ readme/lint:
 
 ## Create README.md by building it from README.yaml
 readme/build: readme/deps $(README_DEPS)
-	@gomplate --file $(README_TEMPLATE_FILE) \
-		--out $(README_FILE) 
+	@gomplate --file $(README_TEMPLATE_FILE) --out $(README_FILE) --config $(BUILD_HARNESS_PATH)/configs/gomplate.yaml
```

Comment:
Does `--config` override `GOMPLATE_CONFIG`?

Reply:
yes, tested this